test/e2e/test/beat/checks.go

```diff
@@ -87,3 +89,38 @@ func checkEvent(url string, check func(int) error) ValidationFunc {
 		return check(len(results.Hits.Hits))
 	}
 }
+
+func containsDocuments(esClient client.Client, indexPattern string) error {
```

Comment:
This is looks like a copy of existing code. I think we should be able to reuse the existing check steps for monitoring in the `checks` package. You just have to expose a new reduced step list for Beats without the side car checks.

Reply:
Understood.  I've implemented a similar exposed func in the `checks` package that just returns the only step needed for beats.  Lmk if this is what you intended.